engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41500to41510.java

```diff
@@ -69,8 +66,6 @@ public InputStream[] getPrepareScripts() {
 
     @Override
     public void performDataMigration(Connection conn) {
-        correctGuestOsNames(conn);
-        updateGuestOsMappings(conn);
```

Comment:
I'm not sure about these schema changes, what would happen to templates/VMs created from existing guest OS on previous versions?